File: piedra-papel-tijera/piedrapapeltijera.js
let jugador = 0
let pc = 0
jugador = prompt("Elige: 1 para piedra, 2 para papel, 3 para tijera")

if(jugador == pc) {
    alert("Es un empate")
} else if(jugador - pc == 1  || jugador - pc == -2) {
    alert(("TU GANAS🎉"))
} else{
    alert("TU PIERDES☹️")
}

File: piedra-papel-tijera/piedrapapeltijera.py
# Juego básico de Piedra, Papel y Tijera
usuario = input("Elige piedra, papel o tijera: ").lower()
computadora = "piedra"  # Por ahora, fijo

# Comparamos las opciones
if usuario == computadora:
    print("Empate")
elif (
    (usuario == "piedra" and computadora == "tijera") or
    (usuario == "papel" and computadora == "piedra") or
    (usuario == "tijera" and computadora == "papel")
):
    print("¡Ganaste!")
else:
    print("Perdiste")
    